tempel.el

```diff
@@ -133,6 +133,10 @@ If a file is modified, added or removed, reload the templates."
 (defvar tempel--inhibit-hooks nil
   "Inhibit tempel modification change hooks from running.")
 
+(defvar tempel--pending-field-modified nil
```

Comment:
Can we find a better solution without this global variable?

Reply:
I also find it ugly but I couldn't, because the modification hook is called twice. I tried removing one of the hooks but this would end up making the field not apply edits from one of the sides.

Reply:
Yes, but maybe one could remove the hook temporarily. See also `tempel--inhibit-hooks` which we could maybe reuse?

Reply:
Note that to make the examples with adjacent fields work properly the modification hook of the second field also needs to be suppressed, even if they are not empty, because otherwise the second fields thinks it has been modified.

Reply:
@minad how to distinguish between the first invocation and the second invocation? if the first invocation is ignored then the overlay is still empty in the second

Reply:
I think it's not possible to prevent double calls to the hook without some global variable, and if every modification is not handled exactly once the modification range passed as argument for the subsequent calls is invalid and can potentially make a mess.